### DISCUSSION TITLE
你们都是几抽出的艾莲？

### DISCUSSION THREAD

**Discussion by sishui-buyin** [share121/inter-knot#842] [其他]
![113829286](https://github.com/user-attachments/assets/76ecca43-3bf3-4235-b14c-a994c5a86989)
你们都是几抽出的艾莲？我不多45抽，但熬了好几天，可以说没有艾莲小姐的日子如地狱

**Comment by MRScongZZZ** [comment]
150抽 准备把新艾丽都炸成灰

**Comment by ShengWangLOG** [comment]
第一发10连

**Comment by FIREFLY26710-WEB** [comment]
30抽3只（天真有邪）

**Comment by KCUB6** [comment]
我保底出的

**Comment by minyue-yu** [comment]
小保底

**Comment by 1451374815** [comment]
歪了

**Comment by wushanef** [comment]
2抽和七抽

**Comment by g98054** [comment]
144抽

**Comment by SSTV-10068** [comment]
额，1抽出丽莎，30多抽出的艾莲

**Comment by xibozhi** [comment]
小保底吃满（悲

**Comment by SonwNight3089** [comment]
吃满160，心灰意冷

**Comment by doufeathers** [comment]
70抽左右

**Reply by zxc123qwer** [reply]
别说了，170发

**Reply by Ellen666design** [reply]
151抽，充的648出的

**Comment by ShengWangLOG** [comment]
还不错的



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月10日(星期六) 晚上11:21
收件人: "share121/inter-knot";
抄送: "Comment";
主题: Re: [share121/inter-knot] 你们都是几抽出的艾莲？ (Discussion #842)





 
70抽左右
 
&mdash;

**Comment by ShengWangLOG** [comment]
这是真倒霉 



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月10日(星期六) 晚上10:35
收件人: "share121/inter-knot";
抄送: "Comment";
主题: Re: [share121/inter-knot] 你们都是几抽出的艾莲？ (Discussion #842)





 
吃满160，心灰意冷
 
&mdash;

**Comment by ShengWangLOG** [comment]
你！不许玩zzz！



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月10日(星期六) 晚上8:43
收件人: "share121/inter-knot";
抄送: "Comment";
主题: Re: [share121/inter-knot] 你们都是几抽出的艾莲？ (Discussion #842)





 
额，1抽出丽莎，30多抽出的艾莲
 
&mdash;

**Comment by Elysia325** [comment]
170两只

**Comment by likeseele** [comment]
不才，2抽艾莲

**Reply by llxcx4** [reply]
你，不准玩ZZZ(红温了)

**Comment by llxcx4** [comment]
我的鲨鱼被小猫吃了( ◢д◣)
